### ISSUE TITLE
Can i use linorobot without depth_sensor?

### ISSUE THREAD

**Issue by fordft168** [linorobot/linorobot2#79]

**Comment by mich1342** [comment]
Hi,

Sure you could do that. I have done it once. Just don't call the depth sensor launch and comment all depth sensor related line in both slam and nav config file

**Comment by fordft168** [comment]
and if without the encoder too? cause all my encoder broke haha

**Comment by mich1342** [comment]
Yes, you could do that as long as you have other source of odometry data such as visual odometry to replace the encoders

**Comment by souravchandrasaha** [comment]
> Yes, you could do that as long as you have other source of odometry data such as visual odometry to replace the encoders

I'm trying to run my AMR without depth camera and its not working . Can you share your package @mich1342 Please . that would be great help.

**Comment by fordft168** [comment]
ok im complete the linorobot2 without depth already but, a bit pain cause need to build every thing in pi5
